spec/features/player_choice_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './app'
+
+feature 'player choice' do
+  scenario 'player can choose rock option' do
+    sign_in_and_play
+    click_link 'Rock'
+  end
+  scenario 'player can choose scissors option' do
+    sign_in_and_play
+    click_link 'Scissors'
+  end
+  scenario 'player can choose paper option' do
+    sign_in_and_play
+    click_link 'Paper'
```

Comment:
Is it worth putting what you expect to be displayed on the page when you've clicked these links?